### PR TITLE
set start method of torch.multiprocessing to 'spawn'

### PR DESCRIPTION
**Pull request by John-WL**

### CODE REVIEW

File: scripts/main.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import modules.scripts as scripts
 import sys
 import importlib
+from torch import multiprocessing
+multiprocessing.set_start_method('spawn')
```
Comment (ljleb) [inline scripts/main.py:5]: does this work if you reload the webui without completely restarting?

Comment (ljleb) [reply scripts/main.py:5]: In case another extension calls this function it will blow up too. May want to consider restoring the previous value as well.

Comment (John-WL) [reply scripts/main.py]: you're right that it's not working, I think it's best if we check the value before setting it

File: lib_comfyui/comfyui_adapter.py [hunk 2]
```diff
@@ -39,7 +35,8 @@ def start_comfyui_process(model_queue, install_location):
 
     try:
         sys.path.insert(0, sys_path_to_add)
-        thread = multiprocessing.Process(target=async_comfyui_loader.main, args=(model_queue, install_location), daemon=True)
+        multiprocessing_spawn = multiprocessing.get_context('spawn')
```
Comment (ljleb) [inline lib_comfyui/comfyui_adapter.py:38]: make sure to create the Queue with the context as well

File: lib_comfyui/comfyui_adapter.py [hunk 3]
```diff
@@ -3,16 +3,13 @@
...
-    multiprocessing.set_start_method('spawn')
-except RuntimeError:
-    pass
 from modules import script_callbacks
 from lib_comfyui import async_comfyui_loader, webui_settings
 importlib.reload(webui_settings)
 importlib.reload(async_comfyui_loader)
 
 
+multiprocessing_spawn = multiprocessing.get_context('spawn')
```
Comment (ljleb) [inline lib_comfyui/comfyui_adapter.py:12]: move context inside start function?

**Review by ljleb — APPROVED** [review]

**Pull request merged**